homeassistant/components/switchbot/translations/en.json

```diff
@@ -22,7 +22,18 @@
                 "data": {
                     "address": "Device address"
                 }
+            },
+            "lock_key": {
+            "description": "The {name} device requires encryption key, details on how to obtain it can be found in the [documentation]({url}).",
```

Comment:
```suggestion
            "description": "The {name} device requires encryption key, details on how to obtain it can be found in the documentation.",
```